Jamie_Keng/skillset_challenge.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <center> Statistical Arbitrage with Vine Copulas - Pair Selection </center>\n",
+    "\n",
+    "Author: Jui Lin(Jamie) Keng <br>\n",
+    "Email: jamiekeng1016@gmail.com <br>\n",
+    "LinkedIn : https://www.linkedin.com/in/jui-lin-jamie-keng-3883b6131/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "This notebook demonstrates four different approaches to the pair selection methods described in section three in [1]. The goal is to select three of the most suitable partner stocks given a target stock to form a quadruple. Based on different measures of dependence, each approach yields similar results, surprisingly. <br>\n",
```

Comment:
I'm not sure how to interpret your scatter plot? What insights can we derive from this?

Reply:
Same thing with the PCA plot.

Reply:
Ok, I see the analysis section. It would have been better to add the analysis under the plot. 

I'm not sure that your choice of visualization communicates value w.r.t quadruples.

Reply:
No worry, scatter plot matrix and partner-stock frequency histogram, as well as PCA plot, are all been deleted